apps/buildog/src/app/organizations/[organizationId]/web/page.tsx

```diff
@@ -101,11 +139,18 @@ export default function DropdownMenuCheckboxes() {
         {header === "NameCenterHeader" && <NameCenterHeader />}
 
         {/* Blog card rendering */}
-        <div className="mt-8">
+        <div className="mt-8 flex flex-wrap gap-4 p-10">
           {card === "DefaultCard" && <DefaultCard />}
-          {card === "RichCard" && <RichCard />}
+          {card === "DateCard" && <DateCard />}
           {card === "ImageCard" && <ImageCard />}
         </div>
```

Comment:
We have a problem in this components, when we talked we discuss that need couple of card component here. But you shouldn't add the components in the component file. You can use for loop to show more.

- remove the same components in date-card.tsx, default-card.tsx, image-card.tsx, this components have to  have only their self